test/fixtures.py

```diff
@@ -4,22 +4,22 @@
 import shutil
 import tempfile
 
-import app
-import app.models as models
-import app.views as views
+import app         as sagittariidae
+import app.models  as models
+import app.views   as views
 
 
 @pytest.fixture(scope='function')
 def ws(request):
-    flask_app = app.app.app
+    flask_app = sagittariidae.app.app
```

Comment:
*app app*... That's the sound a sagittariidae makes while filing; like the *meep meep* of the roadrunner. :-)